doc.ddoc

```diff
@@ -1,5 +1,7 @@
 ROOT_DIR =
 ROOT = .
+SELF_PATH =
+BUGZILLA_COMPONENT = websites
```

Comment:
Done - turns out we don't need `BUGZILLA_COMPONENT` now, because we already have `PROJECT` :)